vagrant/lib/forklift/box_factory.rb

```diff
@@ -29,6 +29,17 @@ def add_boxes!(box_file)
       end
     end
 
+    def filter_boxes!
+      box_config = Settings.new.settings['boxes']
+      # rubocop:disable Style/SafeNavigation
```

Comment:
This suggests we should set our Rubocop rules to target 2.0 compliance.

Reply:
probably, yeah.